modules/ROOT/pages/addition/kubernetes/daemonset.adoc

```diff
@@ -0,0 +1,96 @@
+= NOM agent as a Daemonset
+
+A single NOM agent to monitor Neo4j instances on a Node in Kubernetes cluster can be setup automatically as a Daemonset. Since agent Daemonset requires per Node
+configuration, it can be provided as a file on a volume mount. To enable Daemonset deployments of NOM agents, a new NOM agent Docker image is published to https://hub.docker.com/r/neo4j/neo4j-ops-manager-agent/tags[Docker Hub verified repository].
+
+[WARNING]
+====
+An agent Daemon pod can not access all resources that Neo4j pod uses. For example, for disk usage metrics, agent needs access to file system or peristent volumes
+that Neo4j pod uses. These need to be mounted to agent Daemon pod.
```

Comment:
A Daemonset should be able to report Node's memory, cpu usage. Other than that only the disk usage that would be different for Neo4j pod since it can be mounted as an external file system.

Reply:
Great!